databases/week2/homework.sql

```diff
@@ -0,0 +1,102 @@
+--Add a task with these attributes: title, description,
+-- created, updated, due_date, status_id, user_id
+ insert INTO task (title,description, created, 
+    updated, due_date, status_id, user_id);
+--Change the title of a task
+update mydb.task set title='do homework' where id=7;
+
+--Change a task due date
+update mydb.task set due_date='2024-07-10 20:16:15' where id =3;
+--Change a task status
+update mydb.task set status_id='1' where title='do homework';
+--Mark a task as complete
+UPDATE mydb.task set status_id='3' WHERE title='do homework';
+--Delete a task
+DELETE FROM mydb.task WHERE title='do homework';
+
+--part 2 School database
+drop database school;
+CREATE DATABASE School;
+USE School;
+CREATE TABLE `class` (
+  `class_id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `being` DATE NOT NULL,
+  `ends` DATE NOT NULL,
+  PRIMARY KEY (`class_id`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8mb4
+COLLATE = utf8mb4_0900_ai_ci;
+
+CREATE TABLE `student` (
+  `student_id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `email` VARCHAR(100) NOT NULL,
+  `phone` INT NOT NULL,
+  `class_id` INT NOT NULL,
+  PRIMARY KEY (`student_id`),
+  CONSTRAINT `class_id`
+    FOREIGN KEY (`class_id`)
+    REFERENCES`class` (`class_id`)
+    ON UPDATE CASCADE)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8mb4
+COLLATE = utf8mb4_0900_ai_ci;
+
+--Create an index on the name column of the student table.
+CREATE INDEX index_name ON student (name);
+--Add a new column to the class table named status
+-- which can only have the following 
+--values: not-started, ongoing, finished (hint: enumerations).
+ALTER TABLE student ADD COLUMN status 
+ENUM('not-started','ongoing','finished')
+
+--Part 3: More queries
```

Comment:
The queries look good, but I think you needed to use other tables and values, look at the instructions for this part.